### DISCUSSION TITLE
Cuda out of Memory

### DISCUSSION THREAD

**Discussion by AaryamanCodes** [carson-katri/dream-textures#173] [Q&A]
Hi!
I am running into an issue while trying to generate a texture.
I have a GTX 1650 which should be sufficient.
`Error: RuntimeError('CUDA out of memory. Tried to allocate 2.00 MiB (GPU 0; 4.00 GiB total capacity; 3.39 GiB already allocated; 0 bytes free; 3.44 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF')`

**Comment by hsyntnctrk** [comment]
I am also using a GTX 1650 and I have exactly the same problem. Full precision. Tried as low as 64x64 with 10 steps still the same error message. Some help would be very much appreciated.

System:
intel Core i5-10300H
GTX 1650 4gb
16 gigs of ram
Win 10.

Blender 3.3.0
Dream Textures V0.0.6

> Error: RuntimeError('CUDA out of memory. Tried to allocate 20.00 MiB (GPU 0; 4.00 GiB total capacity; 3.41 GiB already allocated; 0 bytes free; 3.46 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF')

**Reply by carson-katri** [reply]
Not much I can do at this point, unfortunately. 4GB is very little VRAM to work with for this.

**Reply by hsyntnctrk** [reply]
thanks a lot for the answer, guess I will have to come back to this when I can afford a better GPU.

**Reply by MegaGlodon** [reply]
Ok I just stumbled upon this reply, @carson-katri so basically the AI and samples fits into the pytorch memory allocation and that seems to be 3.41 gib... That does not leave enough to run windows and also start a process, there is no solution for this ? Can some of the process be split between RAM and VRAM if we accept slower speeds ? Thanks.

**Reply by mj1911** [reply]
I tried to install and run SD itself (not this plug-in) on a machine with a 4GB 750 TI, and got essentially the same message.  My take-away from reading other posts about it is that 10GB VRAM is a good minimum, so a 12GB card is the way to go.  And it will _not run_ on a 4GB card, at all.  Would've been really helpful to know this up-front.
